tools/source-exclude-artifacts.txt

```diff
@@ -0,0 +1,24 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# NOTE:
+# This is a list of artifacts in the repository that should
+#  not be included in source release archives due to licensing
+#  restrictions.
+
+R-package
```

Comment:
What is the concern about the R frontend?

Reply:
It was already excluded in the release runbook (see https://cwiki.apache.org/confluence/display/MXNET/Release+Process#ReleaseProcess-Step1.10.Createartifactsforthereleaseandpushtothedistfolder). Has this licensing issue been resolved? Can it be removed from the exclude list?

Reply:
I don't know why it was listed there. That is very dubious, because in that case R-package has never been an official feature of MXNet and we may just delete the code directly as it's no longer maintained anyways..

Reply:
Verified with @szha, we still need to remove R-package, so this should be correct.